_includes/quick-guide-table.html

```diff
@@ -54,26 +54,26 @@
     <div class="arrow arrow-down"></div>
   </div> 
   <div>
-    Assess whether SDA is the right place for your data. The SDA can only take
-    certain types of data and the data submitted to the repository must require
-    controlled access (i.e. that formal applications must be made to
-    access submitted files). Read about how to <a href="../is-sda-suitable.html">find
-    out if SDA is suitable or not</a>.
+    Assess whether SDA is the right place for your data. The repository is for
```

Comment:
This should probably be "the SDA" right?
```suggestion
    Assess whether the SDA is the right place for your data. The repository is for
```